mmdet/datasets/custom.py

```diff
@@ -88,13 +88,24 @@ def __init__(self,
                 self.proposal_file = osp.join(self.data_root,
                                               self.proposal_file)
         # load annotations (and proposals)
-        with self.file_client.get_local_path(self.ann_file) as local_path:
-            self.data_infos = self.load_annotations(local_path)
+        if hasattr(self.file_client, 'get_local_path'):
+            with self.file_client.get_local_path(self.ann_file) as local_path:
+                self.data_infos = self.load_annotations(local_path)
+        else:
+            self.data_infos = self.load_annotations(self.ann_file)
+            warnings.warn('Do not use `mmcv.FileClient.get_local_path` here, '
```

Comment:
warnings should be put before line 95 and indicate users that `we treat the self.ann_file as local paths and it might cause errors if the path is not a local path`.